.circleci/config.yml

```diff
@@ -322,31 +281,6 @@ jobs:
             - target
           key: cargocache-v2-controllers:1.58.1-{{ checksum "~/project/Cargo.lock" }}
 
-  package_utils:
-    docker:
-      - image: rust:1.58.1
-    working_directory: ~/project/packages/utils
-    steps:
-      - checkout:
-          path: ~/project
-      - run:
-          name: Version information
-          command: rustc --version; cargo --version; rustup --version; rustup target list --installed
-      - restore_cache:
-          keys:
-            - cargocache-v2-utils:1.58.1-{{ checksum "~/project/Cargo.lock" }}
-      - run:
-          name: Build library for native target
-          command: cargo build --locked
-      - run:
-          name: Run unit tests
-          command: cargo test --locked
-      - save_cache:
-          paths:
-            - /usr/local/cargo/registry
-            - target
-          key: cargocache-v2-utils:1.58.1-{{ checksum "~/project/Cargo.lock" }}
-
   package_cw1:
     docker:
       - image: rust:1.58.1
```

Comment:
Independent of this PR, but we should bump to >= 1.59.0

Reply:
Definitely! Will do